lib/fortify/controller.rb

```diff
@@ -18,6 +18,10 @@ def set_fortify_user
       Fortify.user = nil
     end
 
+    def authorize(record)
+      raise NotAuthorizedError unless record.policy.public_send(params[:action] + "?")
+    end
```

Comment:
I like this idea, but how does it work with model-level authorization in practice? Can you specify `can :some_action` in the policy or does a mapping need to exist between controller action and model crud actions?